### PR TITLE
Add contact table on oxygen database

### PR DESCRIPTION
**Pull request by assma8jaballah**

### CODE REVIEW

**Review by LaserHuy — APPROVED** [review]

File: database.sql [hunk 1]
```diff
@@ -103,12 +103,12 @@ CREATE TABLE `Course` (
...
 --
 
-CREATE TABLE 'Discipline' (
-  'id' INT NOT NULL,
-  'icon' VARCHAR(100) NOT NULL,
-  'name' VARCHAR(100) NOT NULL,
-  'description' VARCHAR(5000) NOT NULL,
-  PRIMARY KEY ('id')
+CREATE TABLE `Discipline` (
+  `id` INT NOT NULL,
```
Comment (Benja-san) [inline database.sql:107]: id should be auto increment

**Review by Benja-san — CHANGES_REQUESTED** [review]
Change branch name to avoid put your firstname or lastname on a branch, it should only describe the us you are working on.

File: database.sql (hunk 1 above)
Comment (assma8jaballah) [reply database.sql:107]: it is  already auto increment
****
- AUTO_INCREMENT pour la table `Discipline`
--
ALTER TABLE `Discipline`
  MODIFY `id` INT NOT NULL AUTO_INCREMENT;

Comment (assma8jaballah) [reply database.sql]: I changed that, with deleting all the alter request + add auto increment in all tables

**Review by Benja-san — APPROVED** [review]

**Pull request merged**